### PR TITLE
Increase nesting restrictions on StackUrlHelper from 6 to 20

### PR DESCRIPTION
**Pull request by gargana**
## Overview

This is a fix for #514. It increases the recursive nesting limit to 20.

## Testing/Steps taken to ensure quality

Ran new unit tests.
Ran end to end tests.

### Notes

n/a

### CODE REVIEW

File: taskcat/_cfn/stack_url_helper.py [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
 
 class StackURLHelper:
-    MAX_DEPTH = 6  # Handle at most 6 levels of nesting in TemplateURL expressions
```
Comment (tonynv) [inline taskcat/_cfn/stack_url_helper.py]: +1

File: tests/test_stack_url_helper.py [hunk 2]
```diff
@@ -81,7 +81,7 @@ def test_flatten_template_url_maxdepth(self):
         helper = StackURLHelper()
         with self.assertRaises(Exception) as context:
             helper.flatten_template_url(
-                "{ one { two } { two { three { four { five { six { seven }}}}} }}"
+                "{1{2{3{4{5{6{7{8{9{{{{{{{{{{{{21}}}}}}}}}}}}}}}}}}}}}"
             )
 
         self.assertTrue("Template URL contains more than" in str(context.exception))
```
Comment (tonynv) [inline tests/test_stack_url_helper.py]: +1

**Comment by tonynv** [comment]
This looks good to me when you are ready let merge ti

**Comment by andrew-glenn** [comment]
+1

**Pull request merged**